lib/usdUfe/ufe/UsdUndoSelectAfterCommand.h

```diff
@@ -71,9 +71,11 @@ template <class OTHER_COMMAND> class UsdUndoSelectAfterCommand : public OTHER_CO
 private:
     static void setSelection(const Ufe::Selection& newSelection)
     {
-        if (newSelection.empty())
-            return;
-        Ufe::GlobalSelection::get()->replaceWith(newSelection);
+        if (newSelection.empty()) {
+            Ufe::GlobalSelection::get()->clear();
```

Comment:
Support an empty input selection - it means the selection was empty before running the command.